packages/sdcv/src-sdcv.cpp.patch

```diff
@@ -0,0 +1,18 @@
+diff -uNr sdcv-0.5.3/src/sdcv.cpp sdcv-0.5.3.mod/src/sdcv.cpp
+--- sdcv-0.5.3/src/sdcv.cpp     2020-08-14 18:06:51.000000000 +0800
++++ sdcv-0.5.3.mod/src/sdcv.cpp 2021-07-18 23:30:05.063094239 +0800
+@@ -136,7 +136,13 @@
+             if (stardict_data_dir)
+                 data_dir = stardict_data_dir;
+             else
+-                data_dir = "/usr/share/stardict/dic";
++                data_dir =
++#ifdef __TERMUX__
++                  __TERMUX_PREFIX__
++#else
++                  "/usr"
++#endif
++                  "/share/stardict/dic";
```

Comment:
Everything that you need to do is to change
```
data_dir = "/usr/share/stardict/dic"
```
to
```
data_dir = "@TERMUX_PREFIX@/share/stardict/dic"
```

Please remove unnecessary lines.

Reply:
Done.. That's how it's actually should be done....